renovate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "baseBranches":[
+     "main",
+     "release-v0.20",
+     "release-v0.19",
+     "release-v0.18",
+     "release-v0.17"
+  ],
+  "constraints":{
+     "go":"1.22"
+  },
+  "prConcurrentLimit":3,
+  "groupName":"all dependencies",
+  "groupSlug":"all",
+  "lockFileMaintenance":{
+     "enabled":false
+  },
+  "labels":[
+     "release-note-none"
+  ],
+  "extends":[
```

Comment:
I like this section to be at the top of the configuration. It is not a must.

Reply:
fixed